test/e2e/network/ingressclass.go

```diff
@@ -42,7 +43,7 @@ var _ = SIGDescribe("IngressClass [Feature:Ingress]", func() {
 		cs = f.ClientSet
```

Comment:
conformance tests probably should not have a `[Feature:Ingress]` tag... that prevents it from running in default e2e configs

Reply:
open a separate PR to:
1. remove the feature tag
2. mark the tests that create a default ingressclass or which depend on there being no default ingressclass as `[Serial]`

Reply:
limit this PR to switching to use v1 types and promoting the API CRUD tests

Reply:
opened issue to track #92147